Resources/Prototypes/_Nuclear14/Entities/Mobs/NPCs/pets.yml

```diff
@@ -0,0 +1,18 @@
+#MARK: Haggis
+- type: entity
+  name: Haggis
+  id: N14MobCatHaggis
+  parent: MobCat
+  description: A little rough around the edges, but still cute as can be.
+  components:
+  - type: Sprite
+    sprite: _Nuclear14\Mobs\Pets\cathaggis.rsi
```

Comment:
[ERRO] go.comp.sprite: Unable to load RSI '/Textures/_Nuclear14\Mobs\Pets\cathaggis.rsi'.

slashes the wrong way